server/storage/mvcc/kvstore.go

```diff
@@ -477,6 +477,11 @@ func restoreIntoIndex(lg *zap.Logger, idx index) (chan<- revKeyValue, <-chan int
 				ki.restore(lg, Revision{Main: rkv.kv.CreateRevision}, rev, rkv.kv.Version)
 				idx.Insert(ki)
 				kiCache[rkv.kstr] = ki
+			} else {
+				lg.Warn("historical revision(s) has been compacted", zap.Int64("main", rev.Main), zap.Int64("sub", rev.Sub))
```

Comment:
Why warning? Is the fact that historical revisions have been compacted a problem that that user should know about?

Reply:
Right, let's remove this warning. It's expected behaviour. Also, you will see lots of such warning message when there are lots of deletion operations.